### ISSUE TITLE
Missing credentials for revokeToken method, when there are not.

### ISSUE THREAD

**Issue by smashboy** [reboxer/discord-oauth2#1]
Im facing an issue with revoke method that says there credentials missing. I've checked data multiple times and it is fine. Cant figure out the problem.

![image](https://user-images.githubusercontent.com/30213662/64922298-70fb4780-d7d6-11e9-97cb-93181edc8a6a.png)

**Comment by reboxer** [comment]
```js
if (!credentials || !this.credentials) throw new Error("Missing credentials for revokeToken method");
```
it looks like i messed up this if condition, changing the or `||` to an and `&&` should fix this, i'll try to publish a fix as soon as possible.

**Comment by reboxer** [comment]
Thanks for opening the issue, this is fixed as of 1.3.1
